src/resources/help/en_US/relnotes3.4.0.html

```diff
@@ -34,6 +34,9 @@
 <p><b><font size="4">General features added in version 3.4</font></b></p>
 <ul>
   <li>Users can no longer edit, delete, or export built in color tables. Users must create new color tables to edit, export, and delete them. Color tables are no longer saved when saving state.</li>
+  <li>Color Table tags are now editable from the GUI. Tags can be added and removed from color tables at will. Built-in color tables cannot have tags that they come with removed.</li>
+  <li>Tag changes can now be saved to and loaded from config/session files.</li> 
+  <li>There are new rules for what characters tag names are allowed to contain.</li>
```

Comment:
Its probably worth just saying the rules here or pointing to the docs where those rules are stated. FWIW...

```suggestion
  <li>Color table tags are now restricted to contain only alphanumeric characters plus spaces, dashes, equals, greater-than or less-than characters .</li>
```